oml/transforms/images/albumentations/transforms.py

```diff
@@ -136,6 +136,12 @@ def get_normalisation_resize_albu(im_size: int, mean: TNormParam = MEAN, std: TN
     )
 
 
+def get_clip_transforms(im_size: int) -> albu.Compose:
```

Comment:
let's follow our existing naming schedule for transforms
so, it this case it would be `get_normalisation_resize_albu_clip`

Reply:
btw, don't forget to add it to __all__